### PR TITLE
Push the images to GHCR

### PR DESCRIPTION
**Pull request by Praveen2112**
The MakeFile is updated to push the images to GHCR

### CODE REVIEW

**Review by losipiuk — APPROVED** [review]
Thanks

File: Makefile [hunk 1]
```diff
@@ -183,9 +189,13 @@ $(LATEST_TAGS): %@latest: %/Dockerfile %-parent-check
 
 $(VERSION_TAGS): %@$(VERSION): %@latest
```
Comment (kokosing) [inline Makefile:190]: How does it work? What is refered then by `$^`. Only ` %@latest` or only `%@$(VERSION)` or both?

**Review by kokosing — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -183,9 +189,13 @@ $(LATEST_TAGS): %@latest: %/Dockerfile %-parent-check
...
 	docker tag $(call docker-tag,$^) $(call docker-tag,$@)
```
Comment (kokosing) [inline Makefile:191]: do you need this line? It is called below. (same below)

**Review by ssheikin — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (losipiuk) [reply Makefile:190]: My understanding is that `$^` is `%@latest` and `$@` is `%@$(VERSION)`. Or the other way around (not very important from perspecitve of this PR).

Comment (Praveen2112) [reply Makefile:190]: @losipiuk  is correct. Prerequisites are captured in `$^`.

Comment (kokosing) [reply Makefile:190]: But this it has double `:`. So I read it like: `$(VERSION_TAGS)` is a target that depends on `%@$(VERSION)` that depends on `%@latest`. 

🤯

Comment (Praveen2112) [reply Makefile:190]: It guess we specify static pattern rules. so `${VERSTION_TAGS}` specify the targets to which we have to apply and `%@$(VERSION)` is the target pattern and `%@latest` is prerequisite pattern

Ref : https://www.gnu.org/software/make/manual/html_node/Static-Usage.html#Static-Usage

**Pull request merged**